changelogs/unreleased/256-dylanbossie-rise8

```diff
@@ -0,0 +1 @@
+Update volume_snapshotter to correctly use AAD URI which fixes behavior for Azure Workload Identities
```

Comment:
Please remove this file @dylanbossie-rise8 , its automatically generated via commits, when a release is cut.

Reply:
changelogs/unreleased/ is not automatically generated at least for all other velero repos.

I suggests keeping the changelog here.

Reply:
Ah thanks ! I didn't notice that i just looked at some other commits and MRs good job @dylanbossie-rise8 for noticing that.

Reply:
> changelogs/unreleased/ is not automatically generated at least for all other velero repos.
> 
> I suggests keeping the changelog here.

Ah thanks ! I didn't notice that i just looked at some other commits and MRs good job @dylanbossie-rise8 for noticing that. Side, note did you have any further questions for above ?